src/smartthings/smartapps/smartthings/amazon-echo-custom.src/amazon-echo-custom.groovy

```diff
@@ -1,40 +1,54 @@
 /**
- * Amazon Alexa
+ * Amazon Alexa Custom Skill RC
  *
- * Supports lights, dimmers and thermostats
+ * Supports lights, dimmers, thermostats, and locks
  *
  * Author: SmartThings
  */
 import groovy.transform.Field
```

Comment:
Should we move import statements below header too to avoid the copy paste issues we had in the past?